src/Sulu/Bundle/SecurityBundle/Security/Exception/NoTokenFoundException.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Security\Exception;
+use Sulu\Bundle\SecurityBundle\Entity\User;
+
+/**
+ * This exception is thrown when a token-email for user without a token is requested
+ * @package Sulu\Bundle\SecurityBundle\Security\Exception
+ */
+class NoTokenFoundException extends SecurityException
```

Comment:
The wording doesn't make a difference for me... But it's not that important for me.